src/main/webapp/styles/personalised-recommendations.css

```diff
@@ -94,3 +90,48 @@ input[type='radio']:checked:after {
 .button:hover {    
   background: #77d1ce;
 }
+
+.recommendations-container {
+  width: 50%;
+}
+
+.recommendations-container h2 {
+  background: #303c6c;
+  border-radius: 15px;
+  color: white;
+  font-weight: 600;
+  height: 30px;
+  margin: 40px 20px 10px;
+}
+
+.recommendations-container h4 {
+  background: #38afac;
+  border-radius: 15px;
+  color: white;
+  font-weight: 400;
+  height: 20px;
+  margin: auto;
+  padding: 15px;
+}
+
+#top-recommendation {
```

Comment:
This doesn't seem to be used in your code. The CSS needs to be checked to make sure everything is relevant and used.